clusterctl/clusterdeployer/clusterdeployer_test.go

```diff
@@ -578,10 +643,54 @@ func TestDeleteBasicScenarios(t *testing.T) {
 	}
 }
 
-func generateMachines() []*clusterv1.Machine {
+func generateMasterNode(name string) *clusterv1.Machine {
 	master := &clusterv1.Machine{}
-	master.Name = "test-master"
+	master.Name = name
 	master.Spec.Versions.ControlPlane = "1.10.1"
+	return master
+}
+
+func generateTestMasterMachine(masterNames []string) []*clusterv1.Machine {
+	var masters []*clusterv1.Machine
+	for _, mn := range masterNames {
+		m := generateMasterNode(mn)
```

Comment:
combine with the next line: `masters = append(masters, generateMasterNode(mn))`

Reply:
done.